taskcluster/requirements.in

```diff
@@ -1,5 +1,4 @@
 # For instructions on managing dependencies, see:
 # https://taskcluster-taskgraph.readthedocs.io/en/latest/howto/bootstrap-taskgraph.html
 
-taskcluster-taskgraph >= 7.0.2
```

Comment:
There's a bunch of explicit references to `taskgraph`; you should keep this specified as an explicit dependency.